test/contact-summary.spec.js

```diff
@@ -73,16 +77,24 @@ describe('getContactSummary', () => {
   it('state used for reports and lineage but not contact', async () => Harness.__with__({ Function: function() { return functionStub; } })(async () => {
     const mockContact = { _id: 'foo' };
     await harness.getContactSummary(mockContact);
-    expect(functionStub.args[0]).to.deep.eq([mockContact, [], []]);
+    const slicedArgs = functionStub.args[0].slice(0, 3);
+    expect(slicedArgs).to.deep.eq([mockContact, [], []]);
+    const chtApi = functionStub.args[0][3];
+    assert.isFunction(chtApi.v1.hasPermissions);
+    assert.isFunction(chtApi.v1.hasAnyPermission);
   }));
 
   it('#71 - mocked reports in state are passed to contact-summary', async () => Harness.__with__({ Function: function() { return functionStub; } })(async () => {
     const mockContact = { _id: 'foo', type: 'person' };
     const mockReport = { _id: 'bar', patient_id: mockContact._id };
-
     harness.pushMockedDoc(mockReport);
     await harness.getContactSummary(mockContact);
-    expect(functionStub.args[0]).to.deep.eq([mockContact, [mockReport], []]);
+
+    const slicedArgs = functionStub.args[0].slice(0, 3);
+    expect(slicedArgs).to.deep.eq([mockContact, [mockReport], []]);
+    const chtApi = functionStub.args[0][3];
+    assert.isFunction(chtApi.v1.hasPermissions);
```

Comment:
probably equally as useful to do something simpler like `expect(functionStub.args[0][3].v1).to.be.defined;`

this allows you to avoid importing assert and overasserting on the behaviour of the chtapi obj